PokeAlarm/Manager.py

```diff
@@ -327,18 +326,11 @@ def set_optional_args(self, line):
 
     # Start it up
     def start(self):
-        self.__process = gipc.start_process(
-            target=self.run, args=(), name=self.__name)
+        self.__process = gevent.spawn(self.run)
```

Comment:
If you ever need to off-load this work into a child process again then the way you did that before was the correct one. However, for doing inter-process communication with the child in a way so that the gevent machinery in the parent is not negatively affected the `gipc.pipe()` is supposed to be used.